compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -234,7 +231,7 @@ protected Config() throws IOException {
                 new ObjCBlockPlugin(),
                 new AnnotationImplPlugin(),
                 new LambdaPlugin(),         
-                new ShadowFramePlugin()
+                new DebugInformationPlugin()
```

Comment:
yep, so till you don't do any stack trace generation there is no affect on runtime perfomance